src/views/ManageList.jsx

```diff
@@ -1,7 +1,80 @@
-export function ManageList() {
+import { addItem } from '../api/firebase';
+import { useRef } from 'react';
+
+export function ManageList({ listPath }) {
+	const formRef = useRef();
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+
+		const formData = new FormData(e.target);
```

Comment:
Cool approach here using the FormData api in conjunction with default form/input functionality to access necessary data. You don't see this often in client-side React apps, but totally legit thing to do so that's cool to see.